test/cli_create_test.go

```diff
@@ -465,16 +465,17 @@ func (suite *PouchCreateSuite) TestCreateWithPidsLimit(c *check.C) {
 func (suite *PouchCreateSuite) TestCreateWithNonExistImage(c *check.C) {
 	cname := "TestCreateWithNonExistImage"
 	// we should use a non-used image, since containerd not remove image immediately.
-	image := "docker.io/library/alpine"
-	res := command.PouchRun("create", "--name", cname, image)
+	DelImageForceOk(c, busyboxImage)
+	res := command.PouchRun("create", "--name", cname, busyboxImage)
```

Comment:
maybe we can use `command.PouchRun("create", "--name", cname, busyboxImage).Assert(c, icmd.Success)` here :)

Reply:
make since